From 58c7c0285100c6c26f299571dac784e8495df357 Mon Sep 17 00:00:00 2001
From: Jonathan Kang <jonathankang@gnome.org>
Date: Wed, 3 Aug 2022 10:55:48 +0800
Subject: [PATCH] zypp: update libzypp dependency version

In commit dd1964255, we use the newly introduced
PoolItem::statusReinit() to reset pool items status. Since this is
introduced in version 17.31.0, update libzypp dependency version.

https://bugzilla.suse.com/show_bug.cgi?id=1199895
---
 backends/zypp/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backends/zypp/meson.build b/backends/zypp/meson.build
index 84d5b6fbf..5957926dd 100644
--- a/backends/zypp/meson.build
+++ b/backends/zypp/meson.build
@@ -1,6 +1,6 @@
 add_languages('cpp')
 
-zypp_dep = dependency('libzypp', version: '>=6.16.0')
+zypp_dep = dependency('libzypp', version: '>=17.31.0')
 
 # define if libzypp returns package size in bytes
 zypp_args = []
-- 
2.37.1

